File: assignment 1/enc221-0097/2024.cpp
#include <iostream>
#include <string>

using namespace std;

class Neighbour{
    public:
    string name;
    string gender;
    int age;

    void work(){
        cout<<"She works."<<endl;
    }
    void study(){
        cout<<"she also goes to school."<<endl;
    }
    void display(){
        cout<<"My neighbour is,"<<name<<",she is"<<gender<<"and is"<<age<<"years of age."<<endl;

    }
};
 int main(){
    Neighbour neighbour1;
    neighbour1.name="Sheila";
    neighbour1.gender="female";
    neighbour1.age=26;

    neighbour1.display();
    neighbour1.work();
    neighbour1.study();

    return 0;
}

File: assignment 1/enc221-0097/enc221-0097/2024.py
class human:
    def __init__(self,name,gender,age):
        self.name=name
        self.gender=gender
        self.age=age

    def display(self):
        print("my neighbour is",self.name,"she is a",self.gender,"and is",self.age,"years of age")

    def works(self):
        print(self.name,"goes to work")

    def study(self):
        print(self.name,"goes to school")

neighbour1=human("Sheila","female",25)
neighbour1.display()
neighbour1.works()
neighbour1.study()

